assets/queries/k8s/default_namespace/query.rego

```diff
@@ -3,14 +3,16 @@ package Cx
 CxPolicy[result] {
 	document := input.document[i]
 
+    checkKind(document.kind)
+
 	metadata = document.metadata
 
 	object.get(metadata, "namespace", "undefined") == "undefined"
 
 	result := {
 		"documentId": input.document[i].id,
 		"issueType": "MissingAttribute",
-		"searchKey": sprintf("metadata.name=%s", [metadata.name]),
+		"searchKey": sprintf("metadata.name={{%s}}", [metadata.name]),
 		"keyExpectedValue": sprintf("Namespace is set in document[%d]", [i]),
 		"keyActualValue": sprintf("Namespace is not set in document[%d]", [i]),
```

Comment:
Please use the format of referring to attributes: `metadata.namespace is set/undefined`. Don't forget to apply this to the rest of the occurrences.